pkg/autoscaling/calculation.go

```diff
@@ -103,14 +127,46 @@ func getTiDBInstances() []instance {
 	return []instance{}
 }
 
-// TODO: get total CPU use time through Prometheus.
-func getTotalCPUUseTime(component ComponentType, instances []instance, duration time.Duration) float64 {
-	return 1.0
+func instancesToStrings(instances []instance) []string {
```

Comment:
How about `getAddresses`?

Reply:
We need to convert `instance` to string for querying somehow and using the address for the conversion is a temporary solution